tests/test_analyze.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+from .doppel.cil.bin import analyze as al
```

Comment:
Ah ok now I understand what was going on with the imports!

You do not need to directly import the `analyze` scripts, since they're bundled with the package. I'm totally cool with this file containing a single test (since the main intent of the issue was to set up the skeleton of a testing framework and get Travis building it).

Could you replace it with a test on one of the smaller objects in the project, such as `doppel.DoppelTestError`? (https://github.com/jameslamb/doppel/blob/master/cli/doppel/reporters.py#L8)